bin/setup_dot1x

```diff
@@ -46,18 +46,19 @@ docker exec daq-ntwk freeradius &
 
 echo "freeradius setup"
 
-cmd/faucet faucet_dot1x 6653
-docker exec faucet_dot1x ip addr flush eth0
+cmd/faucet local dot1x 6653
```

Comment:
why local?  Isn't all the dot1x code in the main faucetsdn?

Reply:
The assumption is we are testing local changes. Just made sense to me to do it local. Should I correct it?

Reply:
as per other discussion: this should be for mainline branch not local changes.